gm4_ink_spitting_squid/pack.mcmeta

```diff
@@ -20,12 +20,8 @@
         ],
         "Updated by": [
             [
-                "Bloo",
-                "https://www.twitter.com/Bloo_dev"
-            ],
-            [
-                "SpecialBuilder32",
-                "https://www.twitter.com/SpecialBuilder"
+                "BluePsychoRanger",
+                "https://www.twitter.com/BluPsychoRanger"

```

Comment:
I think we retain previous updaters? I may be wrong though.

Reply:
I'm not sure tbh, if we did that would eventually create a long list tho, right?

Reply:
The website only takes the credits from the last version (master). Updated By is also when you're doing module specific updates. So for example I think this could warrant as such, but updating all the attributes ids to the 1.16 names shouldn't.

Reply:
I think it's okay to overwrite/replace old updated by information in the master. Old versions will still have the credit listed in the pack.mcmeta, just not displayed on the site itself.

Reply:
I guess this is a discussion to be had so we can have a standard way of showing updated-by on the site but in this case Special is ok with it and Bloo probably is too, so we'll just go with that.